docs/source/pages/using.rst

```diff
@@ -251,6 +267,32 @@ Then the analysis can be run with:
 
 With ``path_to_sbom`` is the path to the SBOM you want to use.
 
+-------------------------------------------------------
+Analyzing dependencies using Python virtual environment
+-------------------------------------------------------
+
+Macaron can automatically identify and analyze the dependencies of a Python package if you provide the path to the virtual environment where the package is installed.
+
+Let's say you want to analyze ``django@5.0.6`` and its dependencies. First create a virtual environment and install ``django@5.0.6``:
+
+.. code-block:: shell
+
+  python3.11 -m venv /tmp/.django_venv
+  /tmp/.django_venv/bin/pip install django==5.0.6
+
+
+Then run Macaron as follows:
+
+.. code-block:: shell
+
+  ./run_macaron.sh analyze -purl pkg:pypi/django@5.0.6 --python-venv "/tmp/.django_venv"
+
+Where ``--python-venv`` is the path to virtual environment.
+
+Alternatively, you can create an SBOM for the python package and provide it to Macaron as input as explained :ref:`here <with-sbom>`.
+
+.. note:: We only support Python 3.11 for this feature of Macaron. Please make sure to install the package using this version of Python.
+
```

Comment:
As Python3.11 is a prerequisite for running dependency resolution for PYPI packages, could we add this in https://oracle.github.io/macaron/pages/installation.html#prerequisites as an optional prerequisite and link back to this section?